src/libs/ajax/DatasetBuilder.ts

```diff
@@ -4,6 +4,8 @@ export interface DomainOption {
   kind: 'domain';
   id: number;
   category: string;
+  conceptCount: number;
```

Comment:
Is this expected to be the total number of concepts or the total number of concepts that have at least one participant? Computing the total number of concepts shouldn't be too bad, but computing all the concepts used in the current dataset may be expensive.